cmd/backup/main.go

```diff
@@ -127,6 +129,10 @@ type config struct {
 	EmailSMTPPort              int           `envconfig:"EMAIL_SMTP_PORT" default:"587"`
 	EmailSMTPUsername          string        `envconfig:"EMAIL_SMTP_USERNAME"`
 	EmailSMTPPassword          string        `envconfig:"EMAIL_SMTP_PASSWORD"`
+	WebdavUrl                  string        `envconfig:"WEBDAV_URL"`
```

Comment:
So just `split_words:"true"` instead of `envconfig:"WEBDAV_URL"` ?

Reply:
Exactly, see https://github.com/kelseyhightower/envconfig#struct-tag-support - the cases where it's spelled out are because the Go linter forces you to uppercase some words like SMTP or HTTP or so where this stops working (I think?)

Reply:
ah nice! (:
Resolved with: 8b1e925c0df914048afe77a369b25403ad9f6384